src/py42/exceptions.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, exception, message=None):
             message = u"Failure in HTTP call {}. {}".format(
                 str(exception), response_content
             )
-            debug.logger.error(message)
+            debug.logger.debug(message)
```

Comment:
yeah, this level of logging seems more appropriate because us hitting this line does indicate a problem with `py42` itself.